cmds/boot/boot/boot.go

```diff
@@ -469,8 +418,9 @@ func Localboot() error {
 		if *dryRun {
 			continue
 		}
-		if err := kexec.Reboot(); err != nil {
-			log.Printf("Kexec Reboot %v failed, %v. Sorry", u, err)
+
+		if err := boot.Execute(); err != nil {
+			log.Printf("kexec of %v failed: %v", u, err)
```

Comment:
I like renaming from kexec.Reboot too boot.Execute, and not all systems will use kexec, so maybe change the error message here to point to code:
boot.Execute of %v failed: %v

?

Reply:
done